src/lib.rs

```diff
@@ -72,6 +67,11 @@ use pki_types::{
     PrivatePkcs8KeyDer, PrivateSec1KeyDer, SubjectPublicKeyInfoDer,
 };
 
+mod pemfile;
+#[cfg(feature = "std")]
+pub use pemfile::{read_all, read_one};
+pub use pemfile::{read_one_from_slice, Error, Item};
```

Comment:
Suggest moving this code in a separate commit as well, to make obvious that it just moves this down by 8 lines.